.github/workflows/pulsar-ci.yaml

```diff
@@ -126,6 +128,79 @@ jobs:
         with:
           action: wait
 
+  owasp-dep-check:
+    name: OWASP dependency check
+    runs-on: ubuntu-20.04
+    timeout-minutes: 120
+    needs: [ 'changed_files_job', 'build-and-license-check' ]
```

Comment:
I wonder if this should be post-poned to happen after the integration tests, just so that we don't run this if something fails

Reply:
it makes sense. I modified it. It's better after the integration tests and in parallel with system-tests to make sure contributors will have the result at the same time for the system tests and the owasp check